dll/win32/shell32/CDefView.cpp

```diff
@@ -3503,6 +3571,46 @@ INT CDefView::_FindInsertableIndexFromPoint(POINT pt)
     return nCount;
 }
 
+void CDefView::_HandleStatusBarResize(int nWidth)
+{
+    LRESULT lResult;
+
+    if (m_isParentFolderSpecial)
+    {
+        int nPartArray[] = {-1};
+        m_pShellBrowser->SendControlMsg(FCW_STATUS, SB_SETPARTS, _countof(nPartArray), (LPARAM)nPartArray, &lResult);
+        return;
+    }
+
+    int nFileSizePartLength = 125;
```

Comment:
```suggestion
    const int nFileSizePartLength = 125;
```

Reply:
@HBelusca moment :trollface: `CDefView.cpp(3593): error C3892: 'nFileSizePartLength': you cannot assign to a variable that is const`

Reply:
then i missed the place where it was assigned.